cli/command/image/load.go

```diff
@@ -78,7 +78,8 @@ func runLoad(ctx context.Context, dockerCli command.Cli, opts loadOptions) error
 		if err != nil {
 			return errors.Wrap(err, "invalid platform")
 		}
-		options.Platform = &p
+		// TODO(thaJeztah): change flag-type to support multiple platforms.
+		options.Platforms = append(options.Platforms, p)
```

Comment:
Some follow-ups will come to allow the flag to be used multiple times. The daemon does not  yet support that, but we can already prepare the CLI for it.